Overhaul campaigns docs for 3.22 (#15807)
sourcegraph/sourcegraph: doc/campaigns/how-tos/tracking_existing_changesets.md

<img src="https://storage.googleapis.com/sourcegraph-assets/docs/images/campaigns/tracking_existing_changesets_overview.png" class="screenshot center">

## Importing changesets into a campaign

To track existing changesets in a campaign you add them to the [campaign spec](../explanations/introduction_to_campaigns.md#campaign-spec) under the `importChangesets` property and apply the campaign spec.

The following example campaign spec tracks multiple existing changesets in different repositories on different code hosts:

```yaml
name: track-important-milestone
description: Track all changesets related to our important milestone

importChangesets:
- repository: github.com/sourcegraph/sourcegraph
  externalIDs: [15397, 15590, 15597, 15583, 15806, 15798]
- repository: github.com/sourcegraph/src-cli
  externalIDs: [378, 373, 374, 369, 368, 361, 380]
- repository: bitbucket.sgdev.org/SOUR/vegeta
  externalIDs: [8]
- repository: gitlab.sgdev.org/sourcegraph/src-cli
  externalIDs: [113, 119]
```

> See "[Creating a campaign](creating_a_campaign.md)" on how to create a campaign from the campaign spec.

Reviewer: `on how` reads a little oddly to me. How about:

```suggestion
See "[Creating a campaign](creating_a_campaign.md)" for instructions on creating a campaign from the campaign spec.
```

Author: ah, much better! Thanks!
